website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -353,3 +366,126 @@ registrations](/docs/agent/services#service-definition-parameter-case).
     the listener to be set up. If the port is not free then Envoy will not expose a listener for the path,
     but the proxy registration will not fail.
   - `protocol` `(string: "http")` - Sets the protocol of the listener. One of `http` or `http2`. For gRPC use `http2`.
+
+### Unix Domain Sockets <sup>Beta</sup>
+
+The following examples show additional configuration for Unix domain sockets.
+
+Added in v1.10.0.
+
+To connect to a service via local Unix Domain Socket instead of a
+port, add `local_bind_socket_path` and optionally `local_bind_socket_mode`
+to the upstream config for a service:
+
+<Tabs>
+<Tab heading="HCL">
+
+```hcl
+upstreams = [
+  {
+    destination_name = "service-1"
+    local_bind_socket_path = "/tmp/socket_service_1"
+    local_bind_socket_mode = "0700"
+  }
+]
+```
+
+</Tab>
+<Tab heading="JSON">
+
+```json
+"upstreams": [
+  {
+      "destination_name":  "service-1",
+      "local_bind_socket_path": "/tmp/socket_service_1",
+      "local_bind_socket_mode": "0700"
+  }
+]
+
+</Tab>
+</Tabs>
+
+This will cause Envoy to create a socket with the path and mode
+provided, and connect that to service-1.
+
+The mode field is optional, and if omitted will use the default mode
+for Envoy. This is not applicable for abstract sockets. See
+https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#envoy-v3-api-msg-config-core-v3-pipe
```

Comment:
```suggestion
<https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#envoy-v3-api-msg-config-core-v3-pipe>
```

Is this automatically rendered as a link on the site? If not, adding the angle brackets should do this in markdown.